psycopg/psycopg/_pipeline.py

```diff
@@ -77,9 +77,11 @@ def is_supported() -> bool:
             BasePipeline._is_supported = pq_version >= 140000
         return BasePipeline._is_supported
 
-    def _enter(self) -> None:
+    def _enter_gen(self) -> PQGen[None]:
         if self.level == 0:
             self.pgconn.enter_pipeline_mode()
+        elif self.command_queue:
+            yield from self._sync_gen()
```

Comment:
I assume that, with this hangeset, we can drop this sync? https://github.com/psycopg/psycopg/blob/4e7cf85e94edef99e9135aa3603e5bf6d6f642c2/psycopg/psycopg/connection.py#L906

Reply:
Indeed! Done.